fission/function.go

```diff
@@ -69,40 +74,87 @@ func printPodLogs(c *cli.Context) error {
 	return nil
 }
 
-func getInvokeStrategy(minScale int, maxScale int, executorType string, targetcpu int) fission.InvokeStrategy {
-
-	if maxScale == 0 {
```

Comment:
I'm confused about this old code.  Were we preventing users from setting minScale=0?